apps/browser/src/platform/popup/browser-popout-window.service.ts

```diff
@@ -73,18 +70,14 @@ class BrowserPopoutWindowService implements BrowserPopupWindowServiceInterface {
 
     const popupWindow = await BrowserApi.createWindow(windowOptions);
 
-    if (!singleActionPopoutKey) {
-      return;
-    }
+    await this.closeSingleActionPopout(singleActionPopoutKey);
     this.singleActionPopoutTabIds[singleActionPopoutKey] = popupWindow?.tabs[0].id;
   }
 
   private async closeSingleActionPopout(popoutKey: string) {
     const tabId = this.singleActionPopoutTabIds[popoutKey];
-    if (!tabId) {
-      return;
-    }
-    await BrowserApi.removeTab(tabId);
+
+    tabId && (await BrowserApi.removeTab(tabId));
```

Comment:
Rendering the associated `popoutKey` as a `null` value when this is called is a good improvement. That said, can we please avoid this kind of shorthand conditional when not incorporating some form of variable assignment? It's more verbose to use an `if` statement, but it's clearer in intent.

```suggestion

    if (tabId) {
        await BrowserApi.removeTab(tabId);
    }
```

Reply:
yeah, I have no objection to that  👍